src/msg.fbs

```diff
@@ -148,12 +148,14 @@ table CodeFetchRes {
   media_type: MediaType;
   source_code: string;
   output_code: string; // Non-empty only if cached.
+  source_map: string; // Non-empty only if cached.
 }
 
 table CodeCache {
   filename: string;
   source_code: string;
   output_code: string;
+  source_map: string;
```

Comment:
This is unrelated to your edits, but can you add a TODO above `source_code` here:
```
// TODO source_code, output_code, source_map should be passed 
// as binary [ubyte] instead of as a string. Since JS will decode the 
// flatbuffer into a string anyway, having these be a string means Rust 
// must also decode utf8.
```